### ISSUE TITLE
How can I merge two bands with different sizes using Rioxarray?

### ISSUE THREAD

**Issue by rfcsilva** [corteva/rioxarray#122]
I have two images that I want to combine as follows[![enter image description here][1]][1] 

Expected result:

[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/BN2Fh.png
  [2]: https://i.stack.imgur.com/548in.png

I tried to use the function rioxarray.merge.merge_arrays() in the following way but I got this error that I can't understand:

```python
>>> import rioxarray
>>> from rioxarray import merge
>>> b4_1 = rioxarray.open_rasterio("LC08_L1TP_204030_20200607_20200607_01_RT/LC08_L1TP_204030_20200607_20200607_01_RT_B4.TIF")
>>> b4_2 = rioxarray.open_rasterio("LC08_L1TP_204031_20200607_20200607_01_RT/LC08_L1TP_204031_20200607_20200607_01_RT_B4.TIF")
>>> merged = merge.merge_arrays([b4_1, b4_2], nodata=0.0)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ruben/anaconda3/envs/rs-aCoisa/lib/python3.7/site-packages/rioxarray/merge.py", line 113, in merge_arrays
    **{key: val for key, val in input_kwargs.items() if val is not None},
  File "/home/ruben/anaconda3/envs/rs-aCoisa/lib/python3.7/site-packages/rasterio/merge.py", line 250, in merge
    boundless=False, masked=True, indexes=indexes)
  File "/home/ruben/anaconda3/envs/rs-aCoisa/lib/python3.7/site-packages/rioxarray/merge.py", line 43, in read
    Image.fromarray(data_window[iii]).resize((width, height))
  File "/home/ruben/anaconda3/envs/rs-aCoisa/lib/python3.7/site-packages/PIL/Image.py", line 1905, in resize
    return self._new(self.im.resize(size, resample, box))
ValueError: image has wrong mode
>>> b4_1
<xarray.DataArray (band: 1, y: 7811, x: 7691)>
[60074401 values with dtype=uint16]
Coordinates:
  * band         (band) int64 1
  * y            (y) float64 4.899e+06 4.899e+06 ... 4.665e+06 4.665e+06
  * x            (x) float64 5.082e+05 5.082e+05 ... 7.389e+05 7.389e+05
    spatial_ref  int64 0
Attributes:
    STATISTICS_MAXIMUM:        63430
    STATISTICS_MEAN:           13305.927976843
    STATISTICS_MINIMUM:        0
    STATISTICS_STDDEV:         13401.114806381
    STATISTICS_VALID_PERCENT:  100
    transform:                 (30.0, 0.0, 508185.0, 0.0, -30.0, 4899015.0)
    scale_factor:              1.0
    add_offset:                0.0
    grid_mapping:              spatial_ref
>>> b4_2
<xarray.DataArray (band: 1, y: 7831, x: 7701)>
[60306531 values with dtype=uint16]
Coordinates:
  * band         (band) int64 1
  * y            (y) float64 4.741e+06 4.741e+06 ... 4.506e+06 4.506e+06
  * x            (x) float64 4.71e+05 4.71e+05 4.711e+05 ... 7.02e+05 7.02e+05
    spatial_ref  int64 0
Attributes:
    transform:     (30.0, 0.0, 470985.0, 0.0, -30.0, 4740615.0)
    scale_factor:  1.0
    add_offset:    0.0
    grid_mapping:  spatial_ref
```

What can I do to fix the problem? Or what alternative to I have to do this but still ending up with a xarray.DataArray?

**Comment by snowman2** [comment]
Are you able to provide input data for debugging?

**Comment by rfcsilva** [comment]
https://drive.google.com/drive/folders/106ndQow1ZYFk79v6dJd_Bs9p3H-IaYbp?usp=sharing

**Comment by snowman2** [comment]
Fix in #123
